examples/plot_baby_afq.py

```diff
@@ -0,0 +1,496 @@
+        elif name == "MdLF":
+            for hemi in ['_R', '_L']:
+                pediatric_bundles[name + hemi] = {
+                    'ROIs': [pediatric_templates[name + '_roi1' + hemi],
+                             pediatric_templates[name + '_roi2' + hemi]],
+                    'rules': [True, True],
+                    'cross_midline': False,
+                    # reuse probability map from ILF
+                    'prob_map': prob_map_data[...,
+                                              prob_map_order.index("ILF" + hemi)],
+                    'uid': uid}
+                uid += 1
+        # Default: two ROIs within hemisphere
+        else:
+            for hemi in ['_R', '_L']:
+                pediatric_bundles[name + hemi] = {
+                    'ROIs': [pediatric_templates[name + '_roi1' + hemi],
+                             pediatric_templates[name + '_roi2' + hemi]],
+                    'rules': [True, True],
+                    'cross_midline': False,
+                    'prob_map': prob_map_data[...,
+                                              prob_map_order.index(name + hemi)],
+                    'uid': uid}
+                uid += 1
+
+    return pediatric_bundles
+
+
+# monkey patch
+api.make_bundle_dict = make_pediatric_bundle_dict
+
+##########################################################################
+# Get example data:
+# -----------------
+# **Diffusion dataset**
+#
+# .. note::
+#   The diffusion data used in this example are from the Developing Human
+#   Connectome project (dHCP):
+#
+#   http://www.developingconnectome.org/project/
+#
+#   Acknowledgement: These results were obtained using data made available from
+#   the Developing Human Connectome Project funded by the European Research
+#   Council under the European Union’s Seventh Framework Programme
+#   (FP/2007-2013) / ERC Grant Agreement no. [319456]
+#
+# .. note::
+#   This example assumes:
+#
+#   - A single subject and session
+#
+#   - Subject's session data has been downloaded into
+#
+#     `AFQ_data\\dHCP\\derivatives`
+#
+#   - Subject's session data has been made BIDS compliant
+#
+#     see https://bids.neuroimaging.io
+#
+
+print("Validating data...")
+
+dhcp_home = op.join(afq_home, 'dHCP')
+
+myafq = api.AFQ(# ==== BIDS parameters ====
+                bids_path=dhcp_home,
+                dmriprep='dHCP neonatal dMRI pipeline',
+                # bids_filters={"suffix": "dwi"}, # default
+                # custom_tractography_bids_filters=None, # default
+                # ===== Registration parameters ====
+                # b0_threshold=50, # default
+                # min_bval=None, # default
+                min_bval=1000,  # override
+                # max_bval=None, # default
+                max_bval=1000,  # override
+                reg_template=pediatric_templates['UNCNeo-withCerebellum-for-babyAFQ'],
+                # reg_subject="power_map", # default
+                reg_subject="b0",  # override
```

Comment:
Both of these (`reg_template` and `reg_subject`) should be skull-stripped using a brain-mask (for pilot data, we have that, for dHCP data it also comes with the anatomical pipeline derivatives). We should over-ride the pyAFQ default brain-mask (which is generated using the `median_otsu` algorithm) with the brain-mask that is provided with the respective datasets.